zkclient/zkclient.go

```diff
@@ -1728,7 +1735,7 @@ func _main() error {
 	// setup high and low prio message channels
 	z.scheduler()
 
-	if newIdentity {
+	if foundClientIdentity == false {
```

Comment:
+1 for using `!foundClientIdentity` here.  This is how the rest of our code (talking about btcsuite and decred) is written.  Comparing bools to true or false will also not pass gosimple, if you ever hook that up in travis.